autostart/lxqt-panel_wayland.desktop.in

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
```

Comment:
I get always 2 panels now after login as there is also the `lxqt-panel.desktop` from regular installation in autostart - in some way it ends up  also in `/etc/xdg/autostart/`  with the _wayland name, didn't have time to investigate.

Reply:
It's here, it installs any `*.desktop` file:
https://github.com/lxqt/lxqt-panel/blob/2ad883e068e4babd54467322c2d139716188d09f/autostart/CMakeLists.txt#L1

Reply:
IMO 'panel_wayland.conf.in` could be `panel_kwin.conf.in` and be inside the plasma backend folder, as it's not generic for wayland but compositor-specific.

Reply:
I think it's better to install a single `.desktop` file (plus another or symlink inside autostart folder)
This is because KWin might get confused about which file the panel belongs and it needs to check the one with absolute path and KDE specific entries.
In the meantime we discuss this issue, I will solve the double install problem